fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -781,6 +782,7 @@ private boolean publishPartition(@NotNull Database db, @NotNull TableCommitInfo
                 LOG.info("Removed non-exist table {} from transaction {}. txn_id={}", tableId, txnLabel, txnId);
                 return true;
             }
+            enablePartitionAggregation = table.enablePartitionAggregation();
```

Comment:
If this will be overridden, then what is the point of getting from Config.enable_partition_aggregation

Reply:
> If this will be overridden, then what is the point of getting from Config.enable_partition_aggregation

This feature may have compatibility issues during upgrades/downgrades. Therefore, tables created in older versions will not be automatically enabled after upgrading.